conf/app.routes

```diff
@@ -1,11 +1,12 @@
 # microservice specific routes
 
-->         /hmrc-frontend                   hmrcfrontend.Routes
-GET        /enrol-pt/introduction           uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.LandingPageController.showLandingPage(redirectUrl: String)
+->         /hmrc-frontend           hmrcfrontend.Routes
+GET        /no-pt-enrolment   uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.AccountCheckController.accountCheck(redirectUrl: String)
```

Comment:
Did Nikki come up with this new url? Not sure if we can change on the fly since it will be what other services will be redirecting to.